client/src/client.ts

```diff
@@ -2367,6 +2368,8 @@ export abstract class BaseLanguageClient {
 	private _initializeResult: InitializeResult | undefined;
 	private _outputChannel: OutputChannel | undefined;
 	private _disposeOutputChannel: boolean;
+	private _traceOutputChannel: OutputChannel | undefined;
+	private _disposeTraceOutputChannel: boolean;
```

Comment:
I think we don't need this boolean. Since the creator of the client passes in a channel in the constructor the client never owns the channel and  therefore never disposes it. The channel should be disposed by the extension itself since it created it.